appveyor.yml

```diff
@@ -54,11 +54,10 @@ build_script:
   - mkdir web\upload\download
   - bin\nim c koch
   - koch boot -d:release
-  - koch docs
```

Comment:
The docs are part of the zip, it's not an optional step really.

Reply:
`winrelease` builds the docs as well https://github.com/nim-lang/Nim/blob/devel/koch.nim#L365